### DISCUSSION TITLE
Is there any way to create static path?

### DISCUSSION THREAD

**Discussion by dntks1942** [openTCS/opentcs#41] [Q&A] [answered]
I want to create static route that includes particular steps, instead of automatic one which created by dispatcher. Is there any possible way?

**Comment by swltr** [comment, marked as answer]
The route for a vehicle is computed by an implementation of the `Router` interface. The default implementation computes routes based on distances / travel times and does not support statically configured routes for given pairs of source and destination points. You can, however, integrate your own `Router` implementation. See https://www.opentcs.org/docs/5.6/developer/developers-guide/opentcs-developers-guide.html#_replacing_default_kernel_components for how to integrate your own components.
